packages/plugins/opentelemetry/src/setup.ts

```diff
@@ -0,0 +1,67 @@
+// This must be run first. Node uses patching on implementations to inject telemetry so must run before the packages
```

Comment:
This would not live inside the plugin as it is server setup for otel. 
The way otel is setup is by patching of libraries such as fetch, for this reason it needs to run outside of the plugin and be one imported before most packages.